Pdf/PdfFactory.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace Massive\Bundle\PdfBundle\Pdf;
+
+use Knp\Snappy\GeneratorInterface;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Symfony\Component\Templating\EngineInterface;
+
+/**
+ * The pdf factory will create pdfs.
+ */
+class PdfFactory
+{
+    /**
+     * @var GeneratorInterface
+     */
+    private $generator;
+
+    /**
+     * @var EngineInterface
+     */
+    private $templating;
+
+    /**
+     * @var RequestStack
+     */
+    private $requestStack;
+
+    /**
+     * PdfFactory constructor.
+     *
+     * @param EngineInterface $templating
```

Comment:
remove this because its already outdated

Reply:
depends of if we release a major version or not?

Reply:
hmm not really :) removing the phpdoc is not a BC break and as I said at least you would have to add the `$requestStack` to the phpdoc

Reply:
ah sorry thought we are in the PdfManager and you want to remove the whole PdfManager service